src/CSET/operators/filters.py

```diff
@@ -93,3 +148,79 @@ def filter_multiple_cubes(
             "The constraints don't produce a single cube per constraint."
         ) from err
     return filtered_cubes
+
+
+def generate_mask(
+    mask_field: Union[iris.cube.Cube, iris.cube.CubeList],
+    condition: str,
+    value: float,
+) -> Union[iris.cube.Cube, iris.cube.CubeList]:
+    """Generate a mask to remove data not meeting conditions.
+
+    Parameters
+    ----------
+    mask_field: iris.cube.Cube | iris.cube.CubeList
+        The field to be used for creating the mask.
+    condition: str
+        The type of condition applied, six available options:
+        '==','!=','<','<=','>', and '>='.
+    value: float
+        The value on the other side of the condition.
```

Comment:
Make the ordering a bit clearer.
```suggestion
        The value on the right hand side of the condition.
```